File: Difficulty: Basic/Sorted Array Search/sorted-array-search.py
#User function Template for python3

class Solution:
    ##Complete this function
    def searchInSorted(self,arr, k):
        left = 0
        right = len(arr) -1
        while(left <= right):
            mid = left + (right - left) // 2
            if arr[mid] == k:
                return True
            elif arr[mid] > k:
                right = mid - 1
            else:
                left = mid + 1
        return False

File: Difficulty: Basic/Sorted Array Search/sorted-array-search.c
bool searchInSorted(int* arr, int n, int k) {
    int low=0;
    int high=n-1;
    
    while(low<=high){
       int mid=low+(high-low)/2;
       
       if(arr[mid]==k)
       return true;
       
        else if(arr[mid]<k)
        low=mid+1;
        
        else{
            high=mid-1;
        }
        
    }
    
    return false;
    
       
}
